crates/cast/bin/tx.rs

```diff
@@ -183,6 +183,11 @@ where
             tx.set_nonce(nonce.to());
         }
 
+        tx.input = TransactionInput {
+            input: if tx_opts.input_only_data { None } else { Some(Bytes::new()) },
+            data: if tx_opts.input_only_input { None } else { Some(Bytes::new()) },
+        };
```

Comment:
this feels a bit weird because this now set the input/data field to empty bytes, 
I believe this would technically be equivalent to null as well, just feels odd

Reply:
Yeah, that's right, the reason is I don't want to make too much changes, so I set the fields to empty bytes or null, then we can check empty bytes or null when we building transaction rather than pass additional arguments

https://github.com/chuwt/foundry/blob/94e369e554338a1dc55c48bfeb301d55b260fa90/crates/cast/bin/tx.rs#L319-#L327

Reply:
fixed, add `input_only_input` to CastTxBuilder